webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -747,7 +747,8 @@ class ForeignCountriesEligibilityFailureDisplaySteuerlotseStep(EligibilityFailur
 class ForeignCountriesDecisionEligibilityInputFormSteuerlotseStep(DecisionEligibilityInputFormSteuerlotseStep):
     name = "foreign_country"
     next_step_data_models = [
-        (ForeignCountryEligibility, 'success'),
+        (ForeignCountrySuccessEligibility, 'success'),
+        (ForeignCountryMaybeEligibility, 'maybe'),
```

Comment:
Oh man..I spent an hour on this and couldn't figure out what I did wrong. The test for "if you have valid data and click foreignCounty: yes then return failure step as next step" just kept failing. Yes. Because the SuccessEligibilityData has ForeignCountryEligibilityData as a previous field. And if a previous field returns an error, that is converted into a InvalidEligibilityError - which breaks the whole thing.
So the options I see are 1) not changing this or 2) somehow catching the InvalidEligibilityError, check if it is thrown because of foreign country and then convert it into a normal ValidationError...
Both are not really appealing.

Reply:
We discussed it and decided to do 1) and add a comment to the models